lib/behaves.rb

```diff
@@ -31,4 +36,17 @@ def defined_behaviors(klass)
       raise NotImplementedError, "Expected `#{klass}` to define behaviors, but none found."
     end
   end
+
+  def add_injected_behaviours(klass)
+    injected_behaviours = injected_behaviours(klass)
+    if injected_behaviours
+      self.class_eval &injected_behaviours
+    end
+  end
+
+  def injected_behaviours(klass)
```

Comment:
prefer to rename the method to something like `check_for_injected(klass)`, to keep in standard with `check_for_unimplemented` and also so that `behaves.rb#41` would read a little better.

Reply:
Removed this function, the action of retrieving the injected_behaviors doesn't require its own function. Have refactored it.